{
  try {
    executeMapper(record,1);
  }
 catch (  Exception e) {
    throw new IOException(StringUtils.stringifyException(e));
  }
}
